pkg/client/clientset_generated/clientset/typed/core/v1/pod_expansion.go

```diff
@@ -41,5 +41,5 @@ func (c *pods) Evict(eviction *policy.Eviction) error {
 
 // Get constructs a request for getting the logs for a pod
 func (c *pods) GetLogs(name string, opts *v1.PodLogOptions) *restclient.Request {
-	return c.client.Get().Namespace(c.ns).Name(name).Resource("pods").SubResource("log").VersionedParams(opts, api.ParameterCodec)
+	return c.client.Get().Namespace(c.ns).Name(name).Resource("pods").SubResource("log").VersionedParams(opts, scheme.ParameterCodec)
```

Comment:
Yeah, probably should build the scheme into the restclient.

> You can do that for expediency but it still needs to be a variable / method on the client and this function should get it from there.

I don't want to do that at the moment because client-gen is using scheme.ParameterCodec everywhere: https://github.com/kubernetes/kubernetes/blob/master/pkg/client/clientset_generated/clientset/typed/core/v1/pod.go#L119